### DISCUSSION TITLE
[Feature]: Angular migration guide

### DISCUSSION THREAD

**Discussion by jcimoch** [web-infra-dev/rspack#7804] [Announcements]
### What problem does this feature solve?

Is there any plan to provide migration for angular similary to other already available on docs?

### What does the proposed API of configuration look like?

https://rspack.dev/guide/start/quick-start#migrating-from-existing-projects

**Comment by hardfist** [comment]
see https://x.com/hardfist_1/status/1828805593928237097?s=46&t=SBAO7K2vXowZ7qEb7HmxBQ

**Reply by xkx9431** [reply]
> https://x.com/hardfist_1/status/1828805593928237097?s=46&t=SBAO7K2vXowZ7qEb7HmxBQ

Our product heavily relies on Angular, and we are currently experiencing long build times. We're using a custom Angular builder, but this isn't meeting our needs effectively.
I would like to explore using Rspack as a potential solution to improve our build times. I believe that by working together as a team, we can evaluate whether Rspack meets our requirements and implement it if it proves beneficial.

Your collaboration on this initiative would be greatly appreciated. Anyone have  the same problem ? Let's work together !

Thank you!
